.eslintrc.js

```diff
@@ -11,6 +11,19 @@ module.exports = require('@sumup/foundry/eslint')(
     },
     parserOptions: {
       project: ['./tsconfig.eslint.json']
-    }
+    },
+    overrides: [
+      {
+        files: [
+          '**/__testfixtures__/**/*.input.*',
+          '**/__testfixtures__/**/*.output.*'
+        ],
+        rules: {
+          'import/no-unresolved': 'off',
+          'notice/notice': 'off',
+          '@typescript-eslint/no-unused-vars': 'off'
```

Comment:
Can we use these only in their occurrences instead of just turning it off?

Reply:
Every single test file needs these rules — better to turn them off in a central place.